recipes/blas_openblas/meta.yaml

```diff
@@ -0,0 +1,21 @@
+package:
+  name: blas_openblas
+  version: 1.0.0
+
+build:
+  number: 0
+  track_features:
+    - blas_openblas
+
+test:
+  commands:
+    - conda list blas_openblas
+
+about:
+  home: NA
+  license: NA
+  summary: Metapackage to enable the use of OpenBLAS in an environment.
```

Comment:
It doesn't really *enable* use of OpenBLAS - it just keeps the packages in the system all using OpenBLAS for their blas needs.

I feel like we need a summary page of how this operates - and for the **home** here, link to that explanation.